### PR TITLE
OCPBUGS-25373: Message is not a required field in the CRDs

### PR DESCRIPTION
**Pull request by yuumasato**
#### What type of PR is this?

/kind bug
/kind regression

#### What this PR does / why we need it:
 
- Removes `message` from the required list of CRD fields
  - This fixes errors when upgrading SPO from v0.8.0 to v0.8.1.
  - Related: https://github.com/k8s-operatorhub/community-operators/pull/3580

#### Which issue(s) this PR fixes:
None

#### Does this PR have test?
N/A

#### Does this PR introduce a user-facing change?

```release-note
Remove "message" from the required list of CRD fields to fix issues when upgrading SPO to v0.8.1
```

### CODE REVIEW

File: bundle/manifests/security-profiles-operator.x-k8s.io_rawselinuxprofiles.yaml [hunk 1]
```diff
@@ -123,7 +123,6 @@ spec:
                       type: string
                   required:
                   - lastTransitionTime
-                  - message
```
Comment (saschagrunert) [inline bundle/manifests/security-profiles-operator.x-k8s.io_rawselinuxprofiles.yaml:126]: Those fields are auto-generated. I think we have to change the go code to fix that up.

**Review by saschagrunert — CHANGES_REQUESTED** [review]

File: bundle/manifests/security-profiles-operator.x-k8s.io_rawselinuxprofiles.yaml (hunk 1 above)
Comment (yuumasato) [reply bundle/manifests/security-profiles-operator.x-k8s.io_rawselinuxprofiles.yaml:126]: Is there a way to override the "+required"?
https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Condition

I'm afraid the fix might mean reverting to using self defined `Condition`.

**Comment by yuumasato** [comment]
I'm closing this in favor of https://github.com/kubernetes-sigs/security-profiles-operator/pull/2023

**Pull request closed without merging**